topology/topology.go

```diff
@@ -82,38 +69,24 @@ type Decrypter interface {
 }
 
 type topologyProvider struct {
-	client       minio.Client
-	documentName string
-	bucketName   string
-	decrypter    Decrypter
+	url       string
+	decrypter Decrypter
 }
 
 func (t *topologyProvider) NetworkTopology() (NetworkTopology, error) {
-	ctx := context.Background()
-
-	obj, err := t.client.GetObject(ctx, t.bucketName, t.documentName, minio.GetObjectOptions{})
```

Comment:
I thought that previously we were using files/storage API for fetching the topology.
Are we sure that IPFs will always contain that file? I have doubts about it.

Reply:
We will still use files to store (get the filecoin integration to pin it), but I would much prefer not fetch from that API as I don't trust it that much and it needs many variables which are annoying to explain and share what they are.

Reply:
but the problem is that I was trying couple of times uploading files to IPFS via the storage or files and then was not be able to download it back with IPFS